crates/protocol/interop/src/message.rs

```diff
@@ -28,7 +28,7 @@ sol! {
     /// @param id Encoded Identifier of the message.
     #[derive(Default, Debug, PartialEq, Eq)]
     #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
-    event ExecutingMessage(bytes32 indexed msgHash, MessageIdentifier id);
+    event ExecutingMessage(bytes32 indexed payloadHash, MessageIdentifier identifier);
```

Comment:
pls add this link to docs mentioning smthg about the choice of var names for correct json: https://github.com/ethereum-optimism/optimism/blob/4ba2eb00eafc3d7de2c8ceb6fd83913a8c0a2c0d/op-supervisor/supervisor/types/types.go#L61-L64

Reply:
Done!